src/auction.js

```diff
@@ -498,8 +498,9 @@ function tryAddVideoBid(auctionInstance, bidResponse, afterBidAdded, {index = au
       transactionId: bidResponse.transactionId
     }), 'video');
   const context = videoMediaType && deepAccess(videoMediaType, 'context');
+  const useCacheKey = videoMediaType && deepAccess(videoMediaType, 'useCacheKey');
 
-  if (config.getConfig('cache.url') && context !== OUTSTREAM) {
+  if (config.getConfig('cache.url') && (useCacheKey || context !== OUTSTREAM)) {
```

Comment:
I do think `useCache` would be a better name for the mediaTypes.video setting - and I think this doesn't need to check if context is outstream; let's just allow it on every video adUnit.

Reply:
Just to be clear - I'm suggesting changing this to say "use the cache if `mediaTypes.video.useCache` is true", which should default to true if missing, except for outstream (just to keep current behavior). So that instead of adding a special case on a special case, we could some day potentially remove this distinction entirely (for caching).

Reply:
Should this parameter perhaps be a bidderSettings option?  

If we're setting this value in the mediaTypes.video object and you had multiple bidders within the same outstream adUnit - how would it work if one bidder used the cache approach and one didn't?  The adUnit shares the same setting if it's coming from the mediaTypes.video - so it may use the cache approach when it may not work for a bidder's outstream setup.

Reply:
As far as I can tell, caching means updating a video bid with `videoCacheKey` and `vastUrl` only if it's missing. if a bid comes with a renderer that doesn't know how to use them, they shouldn't hurt? Do bidders care about the cache for the instream case? it seems backwards to me that they would.

Reply:
Bidders I don't think would care about prebid cache since it just happens automatically (from their perspective) or not - as the rendering is handled by the publisher's video player setup.  Side note - Though I'm not sure publishers really would elect to not configure prebid cache if they were running instream, since the docs point that way strongly and its their benefit for the ad experience.  

But outstream is different in only that the bidder has something of a hand to work with the renderer interface we have setup in Prebid.js - to preconfigure the renderer and setup the call that starts the process.  So if the process/workflow changes unexpectedly, that may cause some issues.  It would depend on the situation and the changes are implemented.  It may be fine in the end with what's being planned here, but wanted to raise the thought and ensure it gets checked as part of the review here (to avoid side-effects/edge-cases).

Reply:
@jsnellbaker are you able to see if the typical outstream rendering workflow for the xandr outstream demand and redering library would have issues if this change were to occur?

Reply:
I will try to test it out soon and follow-up with the results from our setup.  Those results may not cover all the outstream setups out there, but it may give some idea.

Reply:
I setup a test page with the flag here enabled and prebid cache url defined in the config.  The outstream ad (though it went through the caching process) rendered on its own fine.  This is because of how the appnexus bidder grabs the video bid information; it uses the raw bid response from the endpoint and passes it to the renderer to process/load onto the page.  So this outstream workflow doesn't utilize the updated vastUrl or videoCacheKey params in the prebid bid object, and thus seems to be okay with these changes (from what I can tell).

Reply:
@renebaudisch It seems like the most popular rendering workflow doesn't take advantage after this change. Is that expected by you? If so, how does this help you? Do you have a different rendering workflow in mind?

Reply:
Hi @patmmccann. I explained our bidflow [above](https://github.com/prebid/Prebid.js/pull/8833#discussion_r948232170).

Also, my intension **is** to not change anything automatically, the change is meant to offer a new option that has to be activated by intension, at least that was my intension. If the decision concludes to always just useCacheKey my case would work either so I won't stop you but also not please you for this, but I just wanted an additional option that I use in my fork to be available in the official build so I do not have to patch my fork on every prebid update we do.

And I haven't testet multiformat adunits, but they should work as before, we do not change the flow here, we just update something inside the video processing so this shouldn't be able to affect native or banners.

Reply:
thanks!